defs.h

```diff
@@ -91,8 +91,14 @@
 #define PAL16_RED		224
 #define PAL16_GRAY		240
 
-#define SCREEN_WIDTH	640
-#define SCREEN_HEIGHT	480
+#define SCREEN_WIDTH	1280

```

Comment:
Did you mean to commit these?

Reply:
not really but they don't hurt I think : P

Reply:
my monitor is 640x480 though, so it won't work :(

Reply:
if my monitor was 640x480, devilutionx wouldn't be my biggest problem :p

Reply:
The default should be vanilla, and also Amiga can't do 640x480 but it does DevilutionX :)

Reply:
My devices (RG300 and RG350) have 3/3.5 inch 320x240 screens.
This resolution will look tiny on them.

Reply:
I don't think the default should change until we have a way to override this via CMake

Reply:
ok restored